examples/cloudml-support-routing/scripts/create_features.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Runs SQL queries to create BigQuery tables for training and prediction."""
+
+import logging
+import os
+import sys
+import utils
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def main():
+  """Runs queries to create training and prediction tables from clean data."""
+
+  # Load config shared by all steps of feature creation.
+  config_path = utils.parse_arguments(sys.argv).config_path
+  config = utils.read_config(config_path)
+  # Project-wide config.
+  global_config = config['global']
+  # Path to SQL files.
+  queries_path = config['file_paths']['queries']
+  # SQL files for different pipeline steps.
+  query_files = config['query_files']
+  # Parameters unique to individual pipeline steps.
+  query_params = config['query_params']
+
+  # Create the dataset to hold data for the pipeline run.
+  utils.create_dataset(
+      destination_project=global_config['destination_project_id'],
+      destination_dataset=global_config['destination_dataset']
```

Comment:
nit: add trailing commas per [go/pystyle#trailing-comma](go/pystyle#trailing-comma)

Reply:
fixed a bunch of these